shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -114,6 +114,15 @@ @implementation FlutterViewController {
   fml::scoped_nsobject<UIScrollView> _scrollView;
   fml::scoped_nsobject<UIView> _keyboardAnimationView;
   MouseState _mouseState;
+  // Timestamp after which a scroll inertia cancel event should be inferred.
+  NSTimeInterval _scrollInertiaEventStartline;
+  // When an iOS app is running in emulation on an Apple Silicon Mac, trackpad input goes through
```

Comment:
It's not catalyst, since the app isn't recompiled. I don't think there is an official name for this runtime, it's exposed by this property: https://developer.apple.com/documentation/foundation/nsprocessinfo/3608556-iosapponmac